### ISSUE TITLE
How to visualize attention map in your paper?

### ISSUE THREAD

**Issue by black12357** [594422814/TransformerTrack#9]
Thank you for your great work? Could you share the code for visualizing attention maps?
Thank you so much!

**Comment by 594422814** [comment]
I simply use the functions such as 'plt.imshow()' and 'plt.savefig()' to print the response maps and attention maps.

**Comment by black12357** [comment]
If it is possible, could you share the code to create attention maps? I try many times to output but still have problems.
Thank you so much.

**Comment by 594422814** [comment]
Response map:
'''
plt.imshow(scores_raw[0,0,:,:].cpu().detach())
res_path = '/home/XXX/debug/frame_' + str(self.frame_num) + '_response_' + '.png'
plt.savefig(res_path)
'''
I use this code in the 'dimp.py' to visualize the response maps

For the attention map, I use a similar way to visualize the 'dot_prod' or 'affinity' in 'ltr/models/target_classifier/multihead_attention.py'.

**Comment by black12357** [comment]
I got it. thank you so much for your help
